lib/sectorbuilder/files.go

```diff
@@ -0,0 +1,88 @@
+package sectorbuilder
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"sync"
+
+	"golang.org/x/xerrors"
+)
+
+func (sb *SectorBuilder) sectorName(sectorID uint64) string {
+	return fmt.Sprintf("s-%s-%d", sb.Miner, sectorID)
+}
+
+func (sb *SectorBuilder) stagedSectorPath(sectorID uint64) string {
+	return filepath.Join(sb.stagedDir, sb.sectorName(sectorID))
+}
+
+func (sb *SectorBuilder) stagedSectorFile(sectorID uint64) (*os.File, error) {
+	return os.OpenFile(sb.stagedSectorPath(sectorID), os.O_RDWR|os.O_CREATE, 0644)
+}
+
+func (sb *SectorBuilder) sealedSectorPath(sectorID uint64) (string, error) {
+	path := filepath.Join(sb.sealedDir, sb.sectorName(sectorID))
+
+	e, err := os.OpenFile(path, os.O_RDWR|os.O_CREATE, 0644)
```

Comment:
why do we want to create the file here?

Reply:
Sectorbuilder doesn't do that for us (78% sure here)